schemas/preferences.sublime-settings.json

```diff
@@ -11,7 +11,19 @@
         "color_scheme": {
           "markdownDescription": "Sets the colors used within the text area.",
           "type": "string",
-          "pattern": "^.+\\.((sublime|hidden)-color-scheme|tmTheme)$",
+          "pattern": "^(.+\\.((sublime|hidden)-color-scheme|tmTheme)|auto)",
+          "default": "Mariana.sublime-color-scheme"
+        },
+        "light_color_scheme": {
+          "markdownDescription": "Controls the `\"color_scheme\"` when set to `\"auto\"`.",
+          "type": "string",
+          "pattern": "^.+\\.((sublime|hidden)-color-scheme|tmTheme)",
+          "default": "Breakers.sublime-color-scheme"
+        },
+        "dark_color_scheme": {
+          "markdownDescription": "Controls the `\"color_scheme\"` when set to `\"auto\"`.",
```

Comment:
I know this is copied from default preferences but I think it should say "... when `color_scheme` is set to "auto""